component/src/test/resources/testng.xml

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change the license header year

Reply:
fixed with https://github.com/wso2-extensions/identity-outbound-auth-reddit/pull/5/commits/2fe50e96466a5573e1188b9c1720d72188ce4f28